src/middlewared/middlewared/plugins/activedirectory.py

```diff
@@ -1138,7 +1138,12 @@ def set_ntp_servers(self):
                                 "Failed to automatically set time source.")
             return
 
-        self.middleware.call_sync('system.ntpserver.create', {'address': pdc, 'prefer': True})
+        try:
+            self.middleware.call_sync('system.ntpserver.create', {'address': pdc, 'prefer': True})
```

Comment:
Hmm, wonder, what if for some reason ntp server is already there?

Reply:
@anodos325 ping

Reply:
Before we try to add PDC emulator we first check for whether the NTP server list is in the default out-of-box state.

Reply:
Ah, I see it now. For the master PR we have to change the default server url